tests/metrics/test_metric_definitions.py

```diff
@@ -338,3 +361,15 @@ def test_ndcg_at_k():
     targets = pd.Series([["a", "b"]])
     result = ndcg_at_k(k=3).eval_fn(predictions, targets)
     assert result.scores[0] == 0.0
+
+
+def test_builtin_metric_call_signature():
+    metric = ndcg_at_k(3)
+    assert set(inspect.signature(metric).parameters.keys()) == {"predictions", "targets"}
```

Comment:
```suggestion
    assert inspect.signature(metric).parameters.keys() == {"predictions", "targets"}
```

can we remove `set`?